so-widgets-bundle.php

```diff
@@ -742,7 +742,7 @@ function load_missing_widget( $the_widget, $class ){
 	 * Add action links.
 	 */
 	function plugin_action_links($links){
-		unset( $links['edit'] );
+		if ( isset( $links['edit'] ) ) unset( $links['edit'] );
```

Comment:
It's a minor thing, but as per [WordPress style guide](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style) we should use braces, even when there's only one line in the block.